Readme.md

```diff
@@ -25,10 +26,26 @@ Include `dist/wpcom.js` in a `<script>` tag:
 <script>
   var wpcom = WPCOM('<your-token>');
   var blog = wpcom.site('your-blog.wordpress.com');
-  blog.posts({ number: 8 }, function(err, list){});
+  blog.posts({ number: 8 }, function(err, list) { ... });
 </script>
 ```
 
+### Authentication
+
+Not all requests require a REST API token.  For example, listing posts on a
+public site is something anyone can do.
+
+If you do need a token, here are some links that will help you generate one:
+- [OAuth2 Authentication](https://developer.wordpress.com/docs/oauth2/)
```

Comment:
The `code style` is for code: library names, functions, etc.  If you want these links to look consistent I'd rather remove the \` from every link there.